content/participants/ttjc@kumargauravsingh14/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Kumar Gaurav Singh
+type: "participant"
+name: "Kumar Gaurav Singh"
+portfolio: "https://malicious-webpage-detection.herokuapp.com/"
+github-link: "https://github.com/kumargauravsingh14"
+linkedin-link: "https://linkedin.com/in/singh-kumar-gaurav"
+looking-for: "job"
+description: "MERN"
+---
+
+Hi! I am Kumar Gaurav Singh. Let me walk you through my intro, projects and blogs.
+
+### introduction
+
+Introvert. INFJ. Don't get me started on anything or else I won't shut up.
+
+## why are you taking this challenge?
+
+I am looking for jobs as a software developer.
+I want to learn more and am looking for a learning circle.
+I am trying to become a good developer and rcritical problem solver.
```

Comment:
Hey! @kumargauravsingh14 a small mistake here:
* rcritical

Reply:
Changed rcirtical to critical